exec/java-exec/src/test/java/org/apache/drill/TestCTASPartitionFilter.java

```diff
@@ -59,48 +58,48 @@ public void withDistribution() throws Exception {
     test("alter session set `planner.slice_target` = 1");
     test("alter session set `store.partition.hash_distribute` = true");
     test("use dfs_test.tmp");
-    test(String.format("create table orders_distribution partition by (o_orderpriority) as select * from dfs_test.`%s/multilevel/parquet`", TEST_RES_PATH));
+    test("create table orders_distribution partition by (o_orderpriority) as select * from dfs_test.`/multilevel/parquet`");
     String query = "select * from orders_distribution where o_orderpriority = '1-URGENT'";
-    testExcludeFilter(query, 1, "Filter", 24);
+    testExcludeFilter(query, 1, "Filter\\(", 24);
```

Comment:
It is no longer sufficient to match "Filter" because the test class name contains "Filter" and the test class name is used to create the tmp directory. And the fully qualified path of a queried file is included in the plan. We want to only match the <b>Filter</b> steps generated in the plan, not the <b>Filters</b> in our file paths. In order to do this I tell it to match "Filter(" which corresponds to a filter step in the plan.